src/services/messagesService.js

```diff
@@ -138,10 +140,28 @@ const postRichObjectToConversation = async function(token, { objectType, objectI
 	})
 }
 
+/**
+ * Updates the last read message id
+ *
+ * @param {string} token The token of the conversation to be removed from favorites
+ * @param {int} lastReadMessage id of the last read message to set
+ */
+const updateLastReadMessage = async function(token, lastReadMessage) {
+	try {
+		const response = await axios.post(generateOcsUrl('apps/spreed/api/v1', 2) + `chat/${token}/read`, {
+			lastReadMessage,
+		})
+		return response
+	} catch (error) {
+		console.error(`Error while updating the last read message to {lastReadMessage}`, error)
```

Comment:
don't catch in services anymore, the front end should do that and handle it gracefully.

Reply:
good point, I forgot to remove after copy-pasting. thanks for the reminder